docs/sources/helm-charts/mimir-distributed/migration-guides/migrate-helm-chart-2.x-to-3.0.md

```diff
@@ -124,13 +125,13 @@ Several parameters that were available in version 2.1 of the mimir-distributed H
 
 ## Example of migrated values
 
-The example values file is compatible with version 2.1 of the mimir-distributed Helm chart, and demonstrates a few things:
+The example values file is compatible with any 2.x version of the mimir-distributed Helm chart, and demonstrates a few things:
 
 - All memcached caches are enabled.
 - The default pod security policy is disabled.
 - ServiceMonitors are enabled.
 - Object storage credentials for block storage are specified directly in the `mimir.config` value.
-  > **Note:** The unmodified parts of the default `mimir.config` are omitted for brevity, even though in a valid 2.1 values file they need to be included.
+  > **Note:** The unmodified parts of the default `mimir.config` are omitted for brevity, even though in a valid 2.x values file they need to be included.
```

Comment:
Please use the admonition syntax at https://grafana.com/docs/writers-toolkit/write/shortcodes/#example.